pom.xml

```diff
@@ -1,16 +1,147 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <groupId>com.epam.tc</groupId>
-    <artifactId>nikita_nechepurenko</artifactId>
-    <version>1.0-SNAPSHOT</version>
-
-    <properties>
-        <maven.compiler.source>14</maven.compiler.source>
-        <maven.compiler.target>14</maven.compiler.target>
-    </properties>
-    
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.epam.tc</groupId>
+  <artifactId>nikita_nechepurenko</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>1.8</java.version>
+    <maven.compiler.source>14</maven.compiler.source>
+    <maven.compiler.target>14</maven.compiler.target>
+    <checkstyle.version>8.42</checkstyle.version>
+    <testng.version>7.4.0</testng.version>
+    <calculator.version>1.0</calculator.version>
+    <testng.configs.path>${project.build.testOutputDirectory}/testngconfigs</testng.configs.path>
+    <testng.suite.file/>
+    <surefire.version>3.0.0-M5</surefire.version>
+    <compiler.version>3.8.1</compiler.version>
+    <commons-lang3.version>3.12.0</commons-lang3.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.epam.tat</groupId>
+      <artifactId>module4</artifactId>
+      <version>${calculator.version}</version>
+      <!--            <scope>system</scope>-->
+      <!--            <systemPath>path_to_jar/calculator-1.0.jar</systemPath>-->
+    </dependency>
+
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <version>${testng.version}</version>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>RELEASE</version>
```

Comment:
`LATEST` or `RELEASE` version can lead to unexpected behavior when not backward compatible or other significant changes occur in dependency